Util/include/Poco/Util/ServerApplication.h

```diff
@@ -232,7 +232,7 @@ class Util_API ServerApplication: public Application
 //
 // Macro to implement main()
 //
-#if defined(_WIN32) && !defined(__MINGW32__)
+#if defined(_WIN32) && !defined(POCO_NO_MINGW_UNICODE)
```

Comment:
No unicode ? I was expected you should have wmain for unicode support?

Reply:
Option `-municode` affect to WinAPI too. For example `InetPton`. It is define to InetPtonA or to InetPtonW. If option `-municode` set, InetPton will be InetPtonW. If not, InetPton will be InetPtonA.
What does mean `A` or `W`:
https://msdn.microsoft.com/ru-ru/library/windows/desktop/dd374089(v=vs.85).aspx

Reply:
As stated in the URL
```
A Windows code page version with the letter "A" used to indicate "ANSI"
A Unicode version with the letter "W" used to indicate "wide"
```